misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -234,13 +296,18 @@ abstract class AnyAssertionsSpec(
         notToBe: Fun1<T?, T?>,
         isSame: Fun1<T?, T?>,
         isNotSame: Fun1<T?, T?>,
-        value: T
+        isNoneOf: Fun2<T?, T?, Array<T?>>,
+        isNotIn: Fun1<T?, Iterable<T?>>,
+        value: T,
+        emptyArray: Array<T?>
```

Comment:
I added an empty Array parameter because I can't instantiate an array inside this function, it needs to be an inline function, if you have any suggestion on how to avoid this please let me know.

Reply:
Fine with me. I guess you could change it to `inline fun <reified T : Any>` and create an empty array inside

Reply:
Unfortunately, Kotlin doesn't support nested inline functions is not supported yet so I can't create an array inside the function.

Reply:
true, local inline functions are not supported